docs/management_api.yml

```diff
@@ -1557,6 +1623,37 @@ definitions:
       log: false
       state: installing
       substate: installing.enter;script:foo-bar
+  DeviceDeployment:
+    type: object
+    properties:
+      id:
+        type: string
+      deployment:
+        $ref: "#/definitions/Deployment"
+      device:
+        $ref: "#/definitions/Device"
+    required:
+      - deployment
+      - device
+    example:
+      id: 0c13a0e6-6b63-475d-8260-ee42a590e8ff
+      deployment:
+        id: 00a0c91e6-7dec-11d0-a765-f81d4faebf6
+        name: production
+        artifact_name: Application 0.0.1
+        status: finished
+        created: 2016-02-11T13:03:17.063493443Z
+        finished: 2016-03-11T13:03:17.063493443Z
+        device_count: 100
+      device:
+        id: 00a0c91e6-7dec-11d0-a765-f81d4faebf6
+        device_type: Raspberry Pi 3
+        status: installing
+        finished: 2016-03-11T13:03:17.063493443Z
+        created: 2016-02-11T13:03:17.063493443Z
+        state: installing
+        substate: installing.enter;script:foo-bar
+        log: false
```

Comment:
nit: The example object is inconsistent. Deployment is finished while device is installing.

Reply:
now it's `inprogress`